src/fiat/bank-authorities.md

```diff
@@ -191,6 +191,34 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7p
 }
 ```
 
+## Verify a bank authority **EXPERIMENTAL**
+
+Verification codes show up on statements when a user makes withdrawals and deposits. To verify an
+account, you need to direct the user to make a topup/withdrawal and then check their statement.
+
+{% endpoint POST https://service.centrapay.com/api/bank-accounts/${bankAccountId}/verify %}
+
+```sh
+curl -X POST "https://service.centrapay.com/api/bank-accounts/WRhAxxWpTKb5U7pXyxQjjY/verify" \
+  -H "x-api-key: 1234" \
+  -H "content-type: application/json" \
+  -d '{ "verificationCode": "1111" }'
+```
+
+**Required Fields**
+
+|      Field       |  Type  |                                                                                                          Description                                                                                                                            |
+| :--------------- | :----- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| verificationCode | String | The code relating to the top up the user created as part of registering a bank account with Centrapay. This code will show up on their bank statement and will be used to verify that the user has access to the bank account being registered. |
```

Comment:
long descriptions in tables should be pulled out into a separate paragraph. In this case this information is already covered in the paragraph above.